src/main.rs

```diff
@@ -64,7 +64,7 @@ fn main() {
     let mut app = App::new("starship")
         .about("The cross-shell prompt for astronauts. ☄🌌️")
         // pull the version number from Cargo.toml
-        .version(crate_version!())
+        .version(shadow::PKG_VERSION)
```

Comment:
It would be nice if we could add something for [`.long_version()`](https://docs.rs/clap/2.33.3/clap/struct.App.html#method.long_version) here so that a curious user could also access this build information, but it's not crucial to getting this merged (I ran into a lifetime issue while trying it, so unless you know what's going on there that seems like it'd be a bit of work to fix).

Reply:
Done.

add `long_version()` value with below code:
```rust

let long_version = shadow::version();
    let mut app = App::new("starship")
        .about("The cross-shell prompt for astronauts. ☄🌌️")
        // pull the version number from Cargo.toml
        .version(shadow::PKG_VERSION)
        .long_version(long_version.as_str())
        // pull the authors from Cargo.toml
        .author(crate_authors!())
        ...
```